lib/locales/hu.ts

```diff
@@ -0,0 +1,938 @@
+    our_lady_of_mount_carmel_mother_and_queen_of_chile: 'Our Lady of Mount Carmel, Mother and Queen of Chile',
+    our_lady_of_perpetual_help: 'Our Lady of Perpetual Help',
+    our_lady_of_sorrows: 'Our Lady of Sorrows',
+    our_lady_of_sorrows_patroness_of_slovakia: 'Our Lady of Sorrows, Patroness of Slovakia',
+    our_lady_of_the_discovery_of_the_hidden_christians: 'Our Lady of the Discovery of the Hidden Christians',
+    our_lady_of_the_gate_of_dawn: 'Our Lady of the Gate of Dawn',
+    our_lady_of_the_miraculous_medal: 'Our Lady of the Miraculous Medal',
+    our_lady_of_the_pillar: 'Our Lady of the Pillar',
+    our_lady_of_the_rosary: 'Our Lady of the Rosary',
+    our_lady_of_the_valley: 'Our Lady of the Valley',
+    our_lady_of_vladimir: 'Our Lady of Vladimir',
+    our_lady_of_walsingham: 'Our Lady of Walsingham',
+    our_lady_queen_of_peace: 'Our Lady of Peace',
+    our_lady_queen_of_poland: 'Our Lady, Queen of Poland',
+    our_lady_refuge_of_sinners: 'Our Lady, Refuge of Sinners',
+    our_lord_jesus_christ_king_of_the_universe: 'Our Lord Jesus Christ, King of the Universe',
+    our_lord_jesus_christ_the_eternal_high_priest: 'Our Lord Jesus Christ, the Eternal High Priest',
+    our_lord_of_miracles: 'Our Lord of Miracles',
+    palm_sunday_of_the_passion_of_the_lord: 'Palm Sunday of the Passion of the Lord',
+    pancras_of_rome_martyr: 'Saint Pancras, Martyr',
+    pantaleon_of_nicomedia_martyr: 'Saint Pantaleon of Nicomedia, Martyr',
+    paschal_baylon_religious: 'Saint Paschal Baylón, Religious',
+    passion_of_saint_john_the_baptist: 'The Passion of Saint John the Baptist',
+    patrick_of_ireland_bishop: 'Saint Patrick, Bishop',
+    patrick_of_ireland_bishop_patron_of_ireland: 'Saint Patrick, Bishop, Patron of Ireland',
+    paul_chen_changpin_and_companions_martyrs: 'Saint Paul Chen Changpin and Companions, Martyrs',
+    paul_liu_hanzuo_priest: 'Saint Paul Liu Hanzuo, Priest and Martyr',
+    paul_miki_and_companions_martyrs: 'Saint Paul Miki and Companions, Martyrs',
+    paul_of_the_cross_priest: 'Saint Paul of the Cross, Priest',
+    paul_of_thebes_hermit: 'Blessed Paul of Thebes, Hermit',
+    paul_vi_pope: 'Saint Paul VI, Pope',
+    paulina_of_the_agonizing_heart_of_jesus_visintainer_virgin: 'Saint Paulina of the Agonizing Heart of Jesus Visintainer, Virgin',
+    paulinus_of_nola_bishop: 'Saint Paulinus of Nola, Bishop',
+    paulinus_of_trier_bishop: 'Saint Paulinus of Trier, Bishop',
+    paulinus_of_york_bishop: 'Saint Paulinus of York, Bishop',
+    pavel_peter_gojdic_bishop: 'Blessed Pavel Peter Gojdič, Bishop and Martyr',
+    pedro_calungsod_martyr: 'Saint Pedro Calungsod, Martyr',
+    pelagius_of_cordoba_martyr: 'Saint Pelagius of Córdoba, Martyr',
+    pentecost_sunday: 'Pünkösdvasárnap',
+    perpetua_of_carthage_and_felicity_of_carthage_martyrs: 'Saints Perpetua and Felicity, Martyrs',*/
+    peter_and_paul_apostles: 'Szent Péter és Pál apostolok',
```

Comment:
According to [_Direktórium 2022_](http://tar.liturgia.hu/Naptar/DIR2022_JAV_2021-12-13.pdf#page=124), it should be `Szent Péter és szent Pál apostolok` (I am not sure about the casing of `szent Pál` vs `Szent Pál` and `apostolok` vs `Apostolok` though). However, the Hungarian translation of the Roman Missal is authoritative and primary source, not the Ordo thus I am not sure if this is actually correct either.